### ISSUE TITLE
executing cucumber-js shows undefined test results

### ISSUE THREAD

**Issue by douglasguerrero** [apickli/apickli#82]
Testing the cucumber example I got undefined test results:

1) Scenario: Setting headers in GET request - features\httpbin.feature:6
   Step: Given I set User-Agent header to apickli - features\httpbin.feature:7
   Message:
     Undefined. Implement with the following snippet:

       Given('I set User-Agent header to apickli', function (callback) {
         // Write code here that turns the phrase above into concrete actions
         callback(null, 'pending');
       });

Apparently when running "cucumber-js features/httpbin.feature" i got undefined results, BUT when I move the features folder to the cucumber library where cucumber.js is and I execute "node cucumber features/httpbin.feature" my tests pass and works fine. Am I doing something wrong here or is this currently an issue?

**Comment by seymen** [comment]
@douglasguerrero - where in the project you are storing apickli-gherkin.js? That file is where the step definitions are implemented.

You need to make sure that node can find the apickli-gherkin module if you are following the readme and have a line like this:
```
module.exports = require('apickli/apickli-gherkin');
```

**Comment by pablochacin** [comment]
I'm experiencing a similar issue. I created the apickli-gherkin.js file under the features/step_definitions folder, as described in the documentation.  However, there's no way cucumber recognize the scenarios:
 
`./node_modules/cucumber/bin/cucumber.js features/myapi.features
`
I' running the local version of cucumber (the one installed as dependency of the test package) because If I run the global cucumber installation, it complains:
```
Error: 
      You appear to be executing an install of cucumber (most likely a global install)
      that is different from your local install (the one required in your support files).
      For cucumber to work, you need to execute the same install that is required in your support files.
      Please execute the locally installed version to run your tests.
```

**Comment by pablochacin** [comment]
it happens that the issue was the cucumber version. apickli is not compatible with cucumber 2 and up due to changes in the setep definition: https://github.com/apickli/apickli/issues/89

**Comment by seymen** [comment]
#106 is coming to fix this

**Comment by seymen** [comment]
fixed by 58a347d4cc62046149ace65dd90d763866f69014
